azurerm/helpers/azure/batch_pool.go

```diff
@@ -0,0 +1,8 @@
+package azure
+
+type BatchPoolScaleMode string
```

Comment:
We don't usually do this for missing constants, any thoughts @tombuildsstuff ?

Reply:
Actually, since I added dedicated blocks for fixed and auto scale these constants (and the scale_mode field) are not required anymore. I test directly on one of the block to be defined to determine what mode should be used for scaling.